torch/jit/__init__.py

```diff
@@ -527,7 +527,7 @@ def _get_valid_constant(v):
 # resolve references to `self.param` or `self.module`.
 
 
-class ScriptMeta(type(torch._C.ScriptModule)):
+class ScriptMeta(type(torch._C.ScriptModule), _ModuleMeta):
```

Comment:
I'm not convinced we want to add this one here. `ScriptModules` are generally for user-defined ones. See my argumentation below